pennylane/tape/tapes/tape.py

```diff
@@ -1014,6 +1040,17 @@ def execute_device(self, params, device):
             params (list[Any]): The quantum tape operation parameters. If not provided,
                 the current tape parameter values are used (via :meth:`~.get_parameters`).
         """
+        obs_wires = [wire for m in self.measurements for wire in m.wires if m.obs is not None]
```

Comment:
Since we seem to be calculating this in so many places, perhaps it's worth storing as an attribute during queue processing.

Reply:
Thanks, done!